checker/tests/nullness/flow/DeadBranch.java

```diff
@@ -0,0 +1,11 @@
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+class DeadBranch {
+
+    Object foo(@Nullable Object myObj, boolean x) {
```

Comment:
Why do we need `boolean x`?

Reply:
Thanks for pointing out! I need to add another case using `x` as the condition.